src/Component/Customer/CustomerSelector.php

```diff
@@ -66,9 +76,9 @@ public function get()
         $customer = null;
         $user = null;
 
-        if (true === $this->securityContext->isGranted('IS_AUTHENTICATED_FULLY')) {
+        if (true === $this->authorizationChecker->isGranted('IS_AUTHENTICATED_FULLY')) {
```

Comment:
This method only returns boolean, no need to compare it with a boolean to get the final boolean